src/parameters/TerminalVelocity.jl

```diff
@@ -116,6 +115,9 @@ Base.@kwdef struct SB2006VelType{FT} <: TerminalVelocityType{FT}
     aR::FT
     bR::FT
     cR::FT
+    ρw::FT
+    ν_air::FT
```

Comment:
Would it make sense to create two types: one for rain sedimentation parameters and one for cloud? Just in case if we want to use SB2006 scheme with Stokes for cloud sedimentation but Chen 2022 for rain sedimentation?